tigera-operator-1.37.yaml

```diff
@@ -79,3 +83,38 @@ test:
           exit 1
         fi
         echo "Operator help command executed successfully."
+    - name: "Launch operator and check for readiness"
+      uses: test/daemon-check-output
+      with:
+        setup: |
+          k3s server --disable-agent &
+          K3S_PID=$!
+          for i in {1..20}; do
+            if [ -f /etc/rancher/k3s/k3s.yaml ]; then
+              export KUBECONFIG=/etc/rancher/k3s/k3s.yaml
+              if kubectl get nodes >/dev/null 2>&1; then
+                echo "k3s is ready"
+                break
+              fi
+            fi
+            echo "Waiting for k3s to become ready..."
+            sleep 2
+          done
+          kubectl config view --minify --raw > /tmp/kubeconfig.yaml
+          mkdir -p /var/run/secrets/kubernetes.io/serviceaccount
+          echo "default" > /var/run/secrets/kubernetes.io/serviceaccount/namespace
```

Comment:
Was there a blocked when using the `uses: test/kwok/cluster`? Why we particularly spin up a k3s cluster here?

Reply:
ACK - thanks for such a great explanation. Should we have a small comment in the `setup:` to point out why we didn't use the `kwow`? (for future reviewer(s))